README.md

```diff
@@ -3,406 +3,224 @@
-        .expect(developerNameInput.value).contains('Peter', 'input contains text "Peter"');
-});
-```
-
-### No Extra Coding
-
-Write tests without boilerplate code.
-
-* TestCafe automatically waits for page loads and XHRs to complete, as well as for DOM elements to become visible. You do not need to write custom code for that.
-* Test runs are isolated, which means that they do not share cookies, local or session storages. There is nothing to clean up between test runs.
-
-### Descriptive Reports
-
-TestCafe automatically generates full-detailed reports that provide a test run summary and comprehensive information about errors.
-Automatic page screenshots, fancy call sites and call stacks free of TestCafe internals allow you to easily detect error causes.
-
-Use one of [built-in reporters](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/reporters.html) to output test results or [create your own one](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/) to produce custom reports.
-
-![Spec Report](docs/articles/images/spec-report.png)
-
-### Straightforward Continuous Integration
-
-TestCafe is easy to set up on popular Continuous Integration platforms as it allows you to test against various browsers: local, remote, cloud (e.g., [Sauce Labs](https://devexpress.github.io/testcafe/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html)) or headless (e.g. [Nightmare](https://github.com/ryx/testcafe-browser-provider-nightmare)).
-You can also create a custom [browser provider](https://devexpress.github.io/testcafe/documentation/extending-testcafe/browser-provider-plugin/index.html) to add support for a browser or a cloud platform of your choice.
-
-### Other Useful Features
-
-* Automatic screenshots of the tested page taken [whenever the test fails](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-s---screenshots-on-fails) or [triggered from test code](https://devexpress.github.io/testcafe/documentation/test-api/actions/take-screenshot.html).
-* [File upload](https://devexpress.github.io/testcafe/documentation/test-api/actions/upload.html) support.
-* Detecting JavaScript errors on the tested page.
-* [Handling native browser dialogs](https://devexpress.github.io/testcafe/documentation/test-api/handling-native-dialogs.html) (i.e., alert, confirm, prompt, beforeunload ) using custom logic whenever they are invoked during a test run.
-* [Resizing the browser window](https://devexpress.github.io/testcafe/documentation/test-api/actions/resize-window.html) during test execution.
-* [Adjusting the speed](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--speed-factor) of test execution.
+**Easy to include in a continuous integration system**<br/>
+TestCafe is launched from a console, and its reports can be viewed in a CI system's interface
```

Comment:
Change to `You can run TestCafe from a console` ? Since it can be run from a js code as well

Reply:
Yes, let's use your phrase